de.fu_berlin.inf.dpp/test/stf/de/fu_berlin/inf/dpp/stf/test/consistency/EditDuringNonHostAddsProjectTest.java

```diff
@@ -7,20 +7,17 @@
 
 import de.fu_berlin.inf.dpp.stf.client.StfTestCase;
 import de.fu_berlin.inf.dpp.stf.client.util.Util;
-import de.fu_berlin.inf.dpp.stf.shared.Constants.TypeOfCreateProject;
 import org.junit.BeforeClass;
-import org.junit.Ignore;
 import org.junit.Test;
 
-public class NonHostAddsProjectWithHostAcceptDelayedTest extends StfTestCase {
+public class EditDuringNonHostAddsProjectTest extends StfTestCase {
 
   @BeforeClass
   public static void selectTesters() throws Exception {
     select(ALICE, BOB, CARL);
   }
 
   @Test
-  @Ignore("Non-Host adding projects is currently deactivated")
   public void testDelayedHostAccept() throws Exception {
```

Comment:
I recommend you change the test setup a little bit.

1. After Bob starts the sharing he continues to write
2. Alice accept the negotiation and after Alice received the project Alice starts writing too
3. Carl accepts and after Carl finished the negotiation he adds also some text.
4. Everyone stops typing.
5. Compare the results.

Reply:
Done, still works just fine :)